openstack/identity/v2/tenants/results.go

```diff
@@ -69,3 +69,8 @@ func (r tenantResult) Extract() (*Tenant, error) {
 type CreateResult struct {
 	tenantResult
 }
+
+// DeleteResult temporarily contains the response from the Delete call.
+type DeleteResult struct {
+	tenantResult
+}
```

Comment:
Probably just an oversight. I took a quick look at each of those commits (`users.Delete` and `ErrResult`), and it looks like they were made 2 days apart. So the implementation of `users.Delete may have predated the creation of `ErrResult`. In addition, the documentation may have shown a user object in the result of a delete operation. It looks like it should be an `ErrResult` (I haven't tested), so we'll accept a PR for it.